docs/rabbitmqctl.8

```diff
@@ -431,6 +431,37 @@ to the node
 .Qq rabbit@cordelia
 .sp
 .Dl rabbitmqctl rename_cluster_node rabbit@misshelpful rabbit@cordelia
+.Pp
+Note that this command only changes the local database.
+It is also necessary to rename the local database directories,
+and to configure the new node name.
+For example:
+.sp
+.Bl -bullet -compact
+.It
+Stop the node:
+.sp
+.Dl rabbitmqctl stop rabbit@misshelpful
+.sp
+.It
+Rename the node in the local database:
+.sp
+.Dl rabbitmqctl rename_cluster_node rabbit@misshelpful rabbit@cordelia
+.sp
+.It
+Rename the local database directories:
+.sp
+.Dl mv /var/lib/rabbitmq/mnesia/rabbit\\@misshelpful /var/lib/rabbitmq/mnesia/rabbit\\@cordelia
+.Dl mv /var/lib/rabbitmq/mnesia/rabbit\\@misshelpful-rename /var/lib/rabbitmq/mnesia/rabbit\\@cordelia-rename
+.Dl mv /var/lib/rabbitmq/mnesia/rabbit\\@misshelpful-plugins-expand /var/lib/rabbitmq/mnesia/rabbit\\@cordelia-plugins-expand
```

Comment:
I suggest you replace the three `.Dl` by this block:

```
.Bd -literal -offset indent -compact
mv \\
  /var/lib/rabbitmq/mnesia/rabbit@misshelpful \\
  /var/lib/rabbitmq/mnesia/rabbit@cordelia
mv \\
  /var/lib/rabbitmq/mnesia/rabbit@misshelpful-rename \\
  /var/lib/rabbitmq/mnesia/rabbit@cordelia-rename
mv \\
  /var/lib/rabbitmq/mnesia/rabbit@misshelpful-plugins-expand \\
  /var/lib/rabbitmq/mnesia/rabbit@cordelia-plugins-expand
.Ed
```

This way, the command lines are split to remain in the 80-column limit and are more readable.